tobac/tracking.py

```diff
@@ -246,7 +261,11 @@ def linking_trackpy(
     else:
         raise ValueError("method_linking unknown")
 
-        # Filter trajectories to exclude short trajectories that are likely to be spurious
+    # Reset trackpy parameters to default values
+    tp.linking.Linker.MAX_SUB_NET_SIZE = 30
```

Comment:
I think instead of setting these to default values, we should query what they are at the beginning of the run and then set them back to that at the end of the run.

Reply:
Agreed, I changed this.